node/resolvers/releaseDetails.ts

```diff
@@ -7,36 +7,31 @@ interface ReleaseDetailArgs {
 }
 
 export default async (_, args: ReleaseDetailArgs, ctx: ColossusContext): Promise<Release> => {
-  try {
-    const resources = new Resources(ctx)
-    const { appName, cacheId } = args
-    const id = cacheId.split('-')[1]
-    const deployment = await resources.kotoClient.getDeployment(appName, id)
+  const resources = new Resources(ctx)
+  const { appName, cacheId } = args
+  const id = cacheId.split('-')[1]
```

Comment:
Mesmo que a estrutura da lista passada seja bem definida antes acho que o acesso ao index 1 aqui é um número mágico. Eu substituiria com alguma constante que expressasse o porque do acesso a esta lista nesse índice.

Reply:
Agreed, but I didn't change this in this PR.

Reply:
For context: this is an intern's project which is currently broken (doesn't ask for user auth and loads forever), so this is more of a rescue PR. Lot's of things I would change/rewrite given the time :(

Reply:
Nice, changing my vote. Approving this PR.